README.md

```diff
@@ -1,2 +1,56 @@
+![tests](https://github.com/ghga-de/httpyexpect/actions/workflows/unit_and_int_tests.yaml/badge.svg)
+[![PyPI version shields.io](https://img.shields.io/pypi/v/httpyexpect.svg)](https://pypi.python.org/pypi/httpyexpect/)
+[![PyPI pyversions](https://img.shields.io/pypi/pyversions/httpyexpect.svg)](https://pypi.python.org/pypi/httpyexpect/)
+[![Coverage Status](https://coveralls.io/repos/github/ghga-de/httpyexpect/badge.svg?branch=main)](https://coveralls.io/github/ghga-de/httpyexpect?branch=main)
+
 # httpyexpect
-An opinon on how to translate server-side HTTP errors to the client side and tools to implement that opinion.
+An opinon on how to translate HTTP errors from server to the client side and
+the tools for the implement of that opinion.
```

Comment:
```suggestion
An opinionated way to translate server side HTTP errors to the client side.
```
Something in this direction would be more concise and still fit the intention

Reply:
Thanks, yes that is much better.